releases/release-1.31/release-notes/maps/pr-124598-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 124598
+releasenote:
+  text: kubectl describe service and ingress will now use endpointslices instead of
```

Comment:
Needs to be updated.

Reply:
Yeah weird the last part got cut off from the [original PR](https://github.com/kubernetes/kubernetes/pull/124598). I think it should be:
```
`kubectl describe service` and `kubectl describe ingress` will now use endpointslices instead of endpoints.
```